packages/components/src/components/buttons/TooltipButton.tsx/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Icon } from '../../Icon';
+import { Tooltip } from '../../Tooltip';
+import { Button, ButtonProps } from '../Button';
+import { useTheme } from '../../../utils';
+
+const StyledButton = styled(Button)`
+    position: relative;
+    padding: 9px 32px;
```

Comment:
Just to be consistent in case top/bottom `padding` changes on `Button`:

```suggestion
    padding-left: 32px;
    padding-right: 32px;
```

Reply:
af0ef391c5c409353f2c89a8ebd2cac54d6c4aa7